### PR TITLE
Fixes #31281 - Registration Form - Add info about JWT access

### PR DESCRIPTION
**Pull request by stejskalleos**
token

### CODE REVIEW

File: app/views/registration/_form.erb [hunk 1]
```diff
@@ -100,6 +100,11 @@
       <div class='col-md-8'>
         <pre class='ellipsis white-space-normal' id='registration_command'><%= @command %></pre>
         <a class='btn btn-default' onclick="tfm.hosts.copyRegistrationCommand();" ><%= _('Copy to clipboard') %></a>
+        <div>&nbsp;</div>
+        <div class="alert alert-info" role="alert">
+          <span class="pficon pficon-info"></span>
+          <%= _("Note that the command contains the authentication token, which grants access to the Foreman under your account until it expires.") %>
```
Comment (tbrisker) [inline app/views/registration/_form.erb]:
```suggestion
          <%= _("Note: the command contains an authentication token which grants access to your Foreman account until it expires.") %>
```

**Review by tbrisker — COMMENTED** [review]
Thanks @stejskalleos, looks good! just a minor suggestion regarding the wording of the message.

**Comment by stejskalleos** [comment]
> Thanks @stejskalleos, looks good! just a minor suggestion regarding the wording of the message.

Updated

**Review by tbrisker — APPROVED** [review]

**Comment by tbrisker** [comment]
2.3 - b739b8e45

**Pull request merged**